sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveUDFSuite.scala

```diff
@@ -253,7 +253,9 @@ class PairUDF extends GenericUDF {
   )
 
   override def evaluate(args: Array[DeferredObject]): AnyRef = {
+    // scalastyle:off println
     println("Type = %s".format(args(0).getClass.getName))
```

Comment:
This is a big change. I still think a number of these printlns are debug leftovers and can be removed, or in some cases turned into logging. I don't know of a good way to review these other than to just sift through this a few times. I think anything in a main() method or close support of a CLI utility can stay; examples too are probably OK with println. Obviously there are some methods whose job it is to print to the console directly. Everything else, not sure where you would generally allow println.

So, this is one I think can be removed?